api/millenium_patron.py

```diff
@@ -171,7 +171,10 @@ def family_name_match(self, actual_name, supposed_family_name):
         """Does `supposed_family_name` match `actual_name`?"""
         if actual_name is None or supposed_family_name is None:
             return False
-        actual_family_name = actual_name.split(',')[0]
+        if actual_name.find(',') != -1:
+            actual_family_name = actual_name.split(',')[0]
+        else:
+            actual_family_name = actual_name.split(' ')[1]
```

Comment:
This fails on the empty string; it will also fail the "CHER" test where the patron has only one name. Check for a space before you split, similar to how you check for a comma before you split.